pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -37,9 +37,9 @@
 # TODO Do we care about haddock when cross-compiling?
 , doHaddock ? !isCross && (!stdenv.isDarwin || stdenv.lib.versionAtLeast ghc.version "7.8")
 , passthru ? {}
-, pkgconfigDepends ? [], libraryPkgconfigDepends ? [], executablePkgconfigDepends ? [], testPkgconfigDepends ? []
+, pkgconfigDepends ? [], libraryPkgconfigDepends ? [], executablePkgconfigDepends ? [], testPkgconfigDepends ? [], benchPkgconfigDepends ? []
 , testDepends ? [], testHaskellDepends ? [], testSystemDepends ? []
-, benchDepends ? []
+, benchDepends ? [], benchHaskellDepends ? [], benchSystemDepends ? []
```

Comment:
I would prefer the name "benchmark" to be spelled out in those attributes to be consistent with the others. Note that we don't abbreviate "executable" or "library" either.